controller/build.go

```diff
@@ -236,6 +236,8 @@ func PostBuild(c *gin.Context) {
 	build.Started = 0
 	build.Finished = 0
 	build.Enqueued = time.Now().UTC().Unix()
+	build.Event = c.DefaultQuery("event", build.Event)
+	build.Deploy = c.DefaultQuery("deploy_to", build.Deploy)	
 	for _, job := range jobs {
```

Comment:
looks good!

we should probably just make sure that `build.Event` is equal to one of these values
https://github.com/drone/drone/blob/master/model/const.go#L3